### PR TITLE
Move coordinator from meta group member

### PR DESCRIPTION
**Pull request by mychaow**

### CODE REVIEW

File: cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java [hunk 1]
```diff
@@ -1326,50 +1326,21 @@ public void receiveSnapshot(SendSnapshotRequest request) throws SnapshotInstalla
    */
   @Override
   public TSStatus executeNonQueryPlan(PhysicalPlan plan) {
```
Comment (LebronAl) [inline cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java:1328]: Why not delete this method as you have implemented it in Coordinator.

File: cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java [hunk 2]
```diff
@@ -90,6 +91,12 @@
    */
   private MetaGroupMember metaGroupMember;
```
Comment (LebronAl) [inline cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java:92]: As you have abstract coordinator to cooperate with metaGroupMember and dataGroupMember, I think maybe it's better to remove metaGroupMember in ClientServer to decouple clientServer(service logic) and metaGroupMember(internal logic).

File: cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java [hunk 3]
```diff
@@ -203,7 +210,7 @@ protected TSStatus executeNonQueryPlan(PhysicalPlan plan) {
       logger.warn("Illegal plan detected： {}", plan);
       return RpcUtils.getStatus(TSStatusCode.EXECUTE_STATEMENT_ERROR, e.getMessage());
     }
-    return metaGroupMember.executeNonQueryPlan(plan);
+    return coordinator.executeNonQueryPlan(plan);
```
Comment (LebronAl) [inline cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java:213]: update javadoc

**Review by LebronAl — CHANGES_REQUESTED** [review]
Great Work! This PR is going to decouple our service logic and internal logic by coordinator.
I have mentioned some issues, please take a look~

File: cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java (hunk 3 above)
Comment (mychaow) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java:213]: ok

File: cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java (hunk 1 above)
Comment (mychaow) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java:1328]: DataGroupMember use the method, so I have to support the method in metaGroupMember.

File: cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java (hunk 2 above)
Comment (mychaow) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java:92]: ok

File: cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java (hunk 1 above)
Comment (jt2594838) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java:1328]: Where does DataGroupMember use this? It does not seem quite reasonable.

**Review by jt2594838 — CHANGES_REQUESTED** [review]

**Comment by jt2594838** [comment]
I would be grateful if you can also update the document
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=170265889&src=contextnavpagetreemode

File: cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java (hunk 1 above)
Comment (LebronAl) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java:1328]: It seems that we can move  `executeNonQueryPlan` from `RaftService` to `TSDataService` in cluster.thrift

Comment (mychaow) [reply cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java:1328]: ok，I will remove the method.

**Comment by mychaow** [comment]
> I would be grateful if you can also update the document
> https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=170265889&src=contextnavpagetreemode

Yes, I'm glad to do it.

**Review by jt2594838 — APPROVED** [review]

**Review by LebronAl — APPROVED** [review]
LGTM

**Comment by neuyilan** [comment]
LGTM

**Pull request merged**